test/common/utils/formatNumber.test.ts

```diff
@@ -0,0 +1,89 @@
+import {assertEquals} from "@std/assert";
+import {describe, it} from "@std/testing/bdd";
+import {CommonLocales, formatNumber} from "../../../src/utils/format-number";
+
+describe("Correct Format Number Utils", () => {
+    // Use a set to store all common locales.
+    // It is useful to invalidate the test if a new locale is added and not tested.
+    const commonLocales: Set<string> = new Set(Object.values(CommonLocales));
```

Comment:
I've never thought adding a thing like that to be sure to test everything. Nice!!!!